### DISCUSSION TITLE
PHPMyAdmin

### DISCUSSION THREAD

**Discussion by moquette** [pvtl/docker-dev#23] [Ideas]
Simply curious why PHPMyAdmin wasn't included as part of the solution?

**Comment by wireblue** [comment]
Hey @moquette, fair question.

Two reasons:

1. Many of this repo's users prefer to use different apps to connect to MySQL like [TablePlus](https://tableplus.com/), [MySQL Workbench](https://www.mysql.com/products/workbench/) and also PHPMyAdmin.
2. It's super easy to set up your own version of PHPMyAdmin. Just extract it to your Projects folder and [configure the DB connection details](https://github.com/pvtl/docker-dev/blob/master/docs/connections.md#mysql).

So in summary, it wasn't worth the burden of us maintaining a built-in version of PHPMyAdmin.

Cheers

**Reply by moquette** [reply]
Good points!
Thanks!
